### PR TITLE
Update to latest CMake/common

### PR DESCRIPTION
**Pull request by hernando**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -27,25 +27,35 @@ include(GitExternal)
...
+
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_COMPILER_IS_CLANG)
   remove_flag(CMAKE_C_FLAGS -Wshadow)
   remove_flag(CMAKE_CXX_FLAGS -Wshadow)
+  remove_flag(COMMON_C_FLAGS -Wshadow)
+  remove_flag(COMMON_CXX_FLAGS -Wshadow)
 elseif(CMAKE_COMPILER_IS_INTEL)
+  remove_flag(COMMON_CXX_FLAGS -Wshadow)
   remove_flag(CMAKE_CXX_FLAGS -Wshadow)
 endif()
```
Comment (eile) [inline CMakeLists.txt]: get rid of the macro and simply add -Wno-shadow (like in other project)

File: CMakeLists.txt [hunk 2]
```diff
@@ -27,25 +27,35 @@ include(GitExternal)
...
 
-include(FindPackages)
+common_package(MPI REQUIRED)
+common_package(OpenGL)
```
Comment (eile) [inline CMakeLists.txt]: Used for?

File: src/files.cmake [hunk 3]
```diff
@@ -101,42 +100,41 @@ set(MUSIC_C_HEADERS
...
+  ${PROJECT_SOURCE_DIR}/src/music/port.hh
+  ${PROJECT_SOURCE_DIR}/src/music/permutation_index.hh
+  ${PROJECT_SOURCE_DIR}/src/music/runtime.hh
+  ${PROJECT_SOURCE_DIR}/src/music/setup.hh
+  ${PROJECT_SOURCE_DIR}/src/music/sampler.hh
+  ${PROJECT_SOURCE_DIR}/src/music/spatial.hh
+  ${PROJECT_SOURCE_DIR}/src/music/subconnector.hh
+  ${PROJECT_SOURCE_DIR}/src/music/synchronizer.hh
+  ${PROJECT_SOURCE_DIR}/src/music/temporal.hh
+  ${PROJECT_SOURCE_DIR}/src/music/version.hh
```
Comment (eile) [inline src/files.cmake]: Just remove  ${PROJECT_SOURCE_DIR}/

File: CMakeLists.txt (hunk 2 above)
Comment (hernando) [inline CMakeLists.txt]: There's a utility called VisualizeNeurons that needs it.

**Comment by hernando** [comment]
Ping

**Comment by bilgili** [comment]
Is it ok with the latest CMake subtree ? I can not catch the speed of change over there, I dont know the consequences.

**Comment by hernando** [comment]
It was OK, now I'm testing it again.

**Comment by hernando** [comment]
The latest commit builds and installs. Now I'll test whether downstream projects can detect it correctly.

**Comment by hernando** [comment]
Too fast. The installation is not correct. The header files are copied one subdirectory too deep.

**Comment by bilgili** [comment]
Oh :)

**Comment by hernando** [comment]
And one header file is not installed. It's not crucial but I'll try to fix it now.

**Pull request merged**